using_the_compiler/README.md

```diff
@@ -1,136 +1,322 @@
-# Using the compiler
+# Using the Compiler
 
-Once you [install](../installation/README.md) the compiler you will have a `crystal` binary at your disposal.
+Once [installed](../installation/), the Crystal compiler should be available as `crystal` command.
 
-In the next sections a dollar sign (`$`) denotes the command line.
+## Compiling and Running a Command
```

Comment:
I think `Compiling and running at once` was more appropriate as a first title than `Compiling and Running a Command`, which is more generic.